src/test/java/com/openlattice/rehearsal/organization/AssemblerTest.kt

```diff
@@ -536,10 +536,9 @@ class AssemblerTest : MultipleAuthenticatedUsersBase() {
                 setOf(Ace(organization.principal, materializePermissions, OffsetDateTime.MAX))
         )
         permissionsApi.updateAcl(AclData(ptMaterializationAcl, Action.ADD))
+        Thread.sleep(60_000L)
```

Comment:
same questions here as on the other PR -- it would be strongly preferable to avoid these Thread.sleeps if possible.

Reply:
Yeah, we need to wait for the `MaterializePermissionSyncTask` to finish, we can change these calls to explicitly re-materialize the entity set though, because we have the other test to explicitly test, if the background task is doing its thing.

Reply:
This suggest that in general we should have some sort of subscribable event bus for tests.